src/_includes/components/apply-note.njk

```diff
@@ -0,0 +1,21 @@
+<div>
+  <div>Octogons shapes go here</div>
+  <div>
+    <p>When you’re ready to apply we need two things:</p>
+    <div>
+      <h5>1. A completed questionare</h5>
```

Comment:
We shouldn't use `hx` headings for these. I think `strong` would probably be the best semantics we can get. Reason being, you don't want to skip heading levels. Since we use this apply-note in different places on the page, there's no guarantee that `h5` will be the next heading level needed. 

Bit more info here https://www.a11yproject.com/checklist/#headings

> For example, don't jump from a h2 to a h4, skipping a h3 element.

Reply:
Understood!